src/bootstrap/bootstrap.py

```diff
@@ -612,8 +612,14 @@ def download_toolchain(self):
                 self.fix_bin_or_dylib("{}/libexec/rust-analyzer-proc-macro-srv".format(bin_root))
                 lib_dir = "{}/lib".format(bin_root)
                 for lib in os.listdir(lib_dir):
-                    if lib.endswith(".so"):
-                        self.fix_bin_or_dylib(os.path.join(lib_dir, lib))
+                    # .so is not necessarily the suffix, there can be version numbers afterwards.
+                    if ".so" in lib:
```

Comment:
this would (hypothetically) evaluate to True on something unrelated like (making things up) "lib.sodium.png".
Consider `"so" in lib.split('.')` instead

Reply:
i think that's okay